ci/ci_setup.sh

```diff
@@ -14,3 +14,7 @@ rm -rf spec/reports # no stale spec reports from previous executions
 
 # Setup the environment
 rake bootstrap # Bootstrap your logstash instance
+
+# Set up some general options for the rspec runner
+echo "--order rand" > .rspec
+echo "--format CI::Reporter::RSpec" >> .rspec
```

Comment:
is there a reason for not having these options in the test tasks as in https://github.com/elasticsearch/logstash/blob/wip/bundler-move/rakelib/test.rake#L18 ?